src/invidious/views/search.ecr

```diff
@@ -45,3 +134,4 @@
         <% end %>
     </div>
 </div>
+<script src="/js/search.js?v=<%= ASSET_COMMIT %>"></script>
```

Comment:
This should be moved into `<head>`.

Reply:
I'm not sure how to put this in the <head> because that's done in some common template for the head section.

I saw that watch.ecr is doing the same as here, but I don't see other places I can reference to where the js is included in the template.

Reply:
The part inside `<% content_for "header" do %>` gets put into `<head>`.

I didn't write `watch.ecr` but if I had I'd put the scripts into the header. I think having the definitions up top makes the code more readable. But since there's a precedent for putting it into the body it's up to you.